src/home.njk

```diff
@@ -5,17 +5,159 @@ permalink: /
 
 {% extends 'layouts/base.njk' %}
 
+{% block headerContent %}
+<div class="admin-controls">
+    <button class="create-button" id="create-button">
+        <svg role="presentation" class="create-question-svg">
+            <use xlink:href="#create"></use>
+        </svg>
+        New Question
+    </button>
+    <div data-netlify-identity-button></div>
+</div>
+{% endblock %}
+
 {% block content %}
-<h1>{{ title }}</h1>
-<div class="wordle-home">
-    Build inverted wordles that collect people answers to a given question and highlight minority answers.
-    <br/><br/>
-    Available Wordles:
-    <br/>
-    <span class="list-title">{{question.title}}:</span> <a href="/answer/">Submit an answer</a> <a href="/wordle/">View Wordle</a>
+<div class="home">
+    <div class="wordles">
+        <div class="header">
+            <span>Workshop Name</span>
+            <span>Question</span>
+            <span>Word Entries</span>
+            <span>Answers</span>
+            <span>Wordle</span>
+            <span>Last Modified</span>
+            <span></span>
+        </div>
+    </div>
+    <span id="status" role="status"></span>
+
+    {# The model dialog for creating new wordles #}
+    <div class="dialog-backdrop no-scroll">
+        <div id="create-dialog" role="alertdialog" aria-modal="true" aria-labelledby="create_dialog_label" aria-describedby="create_dialog_desc" class="hidden">
+            <h2 id="create_dialog_label" class="dialog_label">Confirm to Create New Question</h2>
+            <div id="create_dialog_desc" class="dialog_desc">
+                <p>
+                    This will create a new webpage. It will take a couple of minutes until links are available. Are you sure you want to proceed?
+                </p>
+            </div>
+            {# The value of the branch name is dynamically set when opening the dialog #}
+            {# aria-controls value is dynamically set to the id of the wordle row that will be created #}
+            <div class="dialog_form_actions">
+                <button type="button" id="create-confirm" aria-controls="" onclick="createWordle(this, 'branchName')">
```

Comment:
The second argument to this function doesn't seem to be used. Would be better namespaced, e.g. something like inverted_wordles.manage.createClicked and separated out into a few functions.